eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -174,11 +179,18 @@ def fitJWST(eventlabel, s4_meta=None):
                 flux_err = meta.lcerr[channel,:]
 
                 # Normalize flux and uncertainties to avoid large flux values
-                flux_err = flux_err/ flux.mean()
-                flux = flux / flux.mean()
+                flux_err /= flux.mean()
+                flux /= flux.mean()
+
+                if meta.testing_S5:
+                    # FINDME: Add exponential ramp while testing
+                    print('****Adding exponential ramp systematic to light curve****')
+                    fakeramp = m.ExpRampModel(parameters=params, name='ramp', fmt='r--')
+                    fakeramp.coeffs = np.array([-1,40,-3, 0, 0, 0])
+                    flux *= fakeramp.eval(time=t_mjdtdb)
 
                 # Load the relevant values into the LightCurve model object
-                lc_model = lc.LightCurve(t_mjdtdb, flux, channel, meta.nspecchan, unc=flux_err, name=eventlabel)
+                lc_model = lc.LightCurve(t_mjdtdb, flux, channel, meta.nspecchan, unc=flux_err, name=eventlabel, time_units=f'MJD_TDB = BJD_TDB - {t_offset}')
```

Comment:
You likely want `int(t_offset)` depending on how `t_offset` gets formatted since it is technically a float